joomla/joomla-cms
# [4.3] Better cache invalidation

Pull Request for Issue #40787 .

### Summary of Changes

If there's a modified_date use it (as unix timestamp) else fallback to the existing mediaversion

<img width="1038" alt="Screenshot 2023-06-19 at 15 55 21" src="https://github.com/joomla/joomla-cms/assets/3889375/7391a2a8-ca28-4412-8c0f-9f35a80716eb">


### Testing Instructions

- Apply the PR
- Edit an image
- check that the rest of the images are loaded from the cache

### Actual result BEFORE applying this Pull Request

Cache is always invalidated

### Expected result AFTER applying this Pull Request

Cache is invalidated only for items/files with different modified date

### Link to documentations
Please select:
